modules/apps/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/AddUADExportProcessesMVCRenderCommand.java

```diff
@@ -61,6 +77,77 @@ public String render(
 	private Portal _portal;
 
 	@Reference
-	private UADApplicationExportHelper _uadApplicationExportHelper;
+	private UADRegistry _uadRegistry;
+
+	private class UADApplicationExport {
```

Comment:
@RenanUmV you do not need an inner class here. Just merging the methods is enough.